web/src/pages/NotificationPage.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <h3 class="text-h3 mb-4">Notifications</h3>
+
+  <AppCard>
+    <v-btn-toggle
+      v-model="showFilter"
+      color="info"
+      group
+      mandatory
+      style="border: 1px #b0b0b0 solid !important; border-radius: 6px"
+      @update:model-value="page = 1"
+    >
+      <v-btn
+        value="All"
+        style="width: 100px; border-left: 1px #b0b0b0 solid"
+        text="All"
+      />
+      <v-btn
+        value="Unread"
+        style="width: 100px; border-left: 1px #b0b0b0 solid"
+        text="Unread"
+      />
+    </v-btn-toggle>
+
+    <v-list
+      class="py-0 mt-4"
+      lines="two"
+    >
+      <v-list-item
```

Comment:
Maybe this should be a DataIterator? See https://vuetifyjs.com/en/components/data-iterators/#usage